src/util.jl

```diff
@@ -1,4 +1,20 @@
 using KernelAbstractions: get_backend, @index, @kernel
+using LoggingExtras
+
+"""
+    logger(fname="WaterLily")
+
+Set up a logger to write the pressure solver data to a logging file named `WaterLily.log`.
+"""
+function logger(fname::String="WaterLily")
+    ENV["JULIA_DEBUG"] = all
+    logger = FormatLogger(fname*".log"; append=false) do io, args
+        (args.level <= Logging.Debug && args.message[1:2]=="ml" ) && print(io, args.message[3:end])
```

Comment:
This could be cleaner with a proper custom logger. For now this is just a hack to have the log file only containing relevant debug informations.